.env-example

```diff
@@ -1,2 +1,5 @@
 THERAPIST_EMAIL_ADDRESS=example@mail.com
 THERAPIST_PASSWORD=supersecretpassword
+DATABASE_URL=postgresql+psycopg2://root:root@db/perfectfit
+RASA_AGENT_URL=http://localhost:5005/webhooks/rest/webhook
+NICEDAY_API_ENDPOINT=http://niceday_api:8080/
```

Comment:
I would suggest to put here the environment variables that will work for local development.

Reply:
Yep, ok I've updated this with exactly what I use in my .env, which works with the docker-compose.yml we provide. I've also added some more info to the README about setting DATABASE_URL.